source/template.yaml

```diff
@@ -1,615 +0,0 @@
-              - "*"
-      VpcConfig:
-        SecurityGroupIds:
-          - !GetAtt ProofResolutionMockSecurityGroup.GroupId
-        SubnetIds:
-          - !Sub
-              - '{{resolve:ssm:/${Environment}/network/subnet/private1/id:1}}'
-              - Environment: !Ref environment
-          -  !Sub
-              - '{{resolve:ssm:/${Environment}/network/subnet/private2/id:1}}'
-              - Environment: !Ref environment
-          - !Sub
-              - '{{resolve:ssm:/${Environment}/network/subnet/private3/id:1}}'
-              - Environment: !Ref environment
-      Tags:
-        environment: !Ref environment
-
-  ProofResolutionMockFunctionArn:
-    Type: AWS::SSM::Parameter
-    Properties:
-      Type: String
-      Name: !Sub
-        - /${Environment}/idp/lambda/ProofResolutionMock/arn
-        - Environment: !Ref environment
-      Value: !GetAtt ProofResolutionMockFunction.Arn
-
-  ProofResolutionMockSecurityGroup:
-    Type: AWS::EC2::SecurityGroup
-    Properties:
-      VpcId: !Sub
-        - '{{resolve:ssm:/${Environment}/network/vpc/id:1}}'
-        - Environment: !Ref environment
-      GroupDescription: Proof Resolution Mock Lambda Function
-      SecurityGroupEgress:
-        - IpProtocol: tcp
-          FromPort: 443
-          ToPort: 443
-          CidrIp: 0.0.0.0/0
-        - IpProtocol: tcp
-          FromPort: 3128
-          ToPort: 3128
-          CidrIp: 172.16.32.0/22
-
-  AWSRubySDKLayer:
-    Type: "AWS::Serverless::LayerVersion"
-    Properties:
-      LayerName: !Sub 
-        - ${Environment}-idp-ruby-sdk
-        - Environment: !Ref environment
-      Description: AWS Ruby SDK
-      ContentUri: aws-ruby-sdk
-      CompatibleRuntimes:
-        - ruby2.7
-      RetentionPolicy: Retain
-    Metadata:
-      BuildMethod: ruby2.7
-
-Outputs:
-  ProofAddressFunction:
-    Description: "Proof Address Lambda Function ARN"
-    Value: !GetAtt ProofAddressFunction.Arn
-  ProofAddressFunctionIamRole:
-    Description: "Proof Address Role created for Proof Address function"
-    Value: !GetAtt ProofAddressFunctionRole.Arn
-
-  ProofAddressMockFunction:
-    Description: "Proof Address Mock Lambda Function ARN"
-    Value: !GetAtt ProofAddressMockFunction.Arn
-  ProofAddressMockFunctionIamRole:
-    Description: "Proof Address Mock Role created for Proof Address Mock function"
-    Value: !GetAtt ProofAddressMockFunctionRole.Arn
-
-  ProofResolutionFunction:
-    Description: "Proof Resolution Lambda Function ARN"
-    Value: !GetAtt ProofResolutionFunction.Arn
-  ProofResolutionFunctionIamRole:
-    Description: "Proof Resolution Role created for Proof Resolution function"
-    Value: !GetAtt ProofResolutionFunctionRole.Arn
-
-  ProofResolutionMockFunction:
-    Description: "Proof Resolution Mock Lambda Function ARN"
-    Value: !GetAtt ProofResolutionMockFunction.Arn
-  ProofResolutionMockFunctionIamRole:
-    Description: "Proof Resolution Mock Role created for Proof Address Mock function"
```

Comment:
Observing that this changes to "...for Proof Resolution Mock function", which seems reasonable given the name of the function.